doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -134,22 +144,21 @@ Admin REST API provides the following actions:
 * [Delete endpoint]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/removeEndpointProfileByKeyHash)
```

Comment:
I don't like this list.
1. Actions listed here relate to at least two fundamentally different entity types: endpoints and schemas. Nonetheless, actions are mixed in a single list.
2. No consistent naming convention. Compare "Get profile based on endpoint key" (what is "endpoint key", anyway???) and "Delete endpoint" (which also works by endpoint ID).